Geometry/HGCalCommonData/src/HGCalGeomParameters.cc

```diff
@@ -384,13 +386,13 @@ void HGCalGeomParameters::loadGeometryHexagon(cms::DDFilteredView& fv,
             double zv[2], rv;
             const std::vector<double>& pars = fv1.parameters();
             if (mode == HGCalGeometryMode::Polyhedra) {
-              zv[0] = pars[4];
-              zv[1] = pars[7];
-              rv = pars[6];
+              zv[0] = pars[3];
```

Comment:
These indices seem almost random. Some comments or more descriptive naming would be helpful to understand what is happening.

Reply:
@kpedro88 It's on my task list to create a shape class for Polyhedra (and some other shapes) to hide these implementation details. Unfortunately, I probably won't get to it soon. In the interim, we have these details here. Some comments saying that this code will be revised might help.

Reply:
I talked to Ianna about this. May be we should discuss about this in some meeting.

Reply:
@ianna Please comment on this issue